pages/k8s/1.32/release-notes.md

```diff
@@ -0,0 +1,126 @@
+
+---
+wrapper_template: templates/docs/markdown.html
+markdown_includes:
+  nav: kubernetes/docs/shared/_side-navigation.md
+context:
+  title: 1.32 Release notes
+  description: Release notes for Charmed Kubernetes
+keywords: kubernetes, release, notes
+tags:
+  - news
+sidebar: k8smain-sidebar
+permalink: 1.32/release-notes.html
+layout:
+  - base
+  - ubuntu-com
+toc: False
+---
+# 1.32
+
+### February 24, 2025 - `charmed-kubernetes --channel 1.32/stable`
+
+The release bundle can also be [downloaded here](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.32/bundle.yaml).
+
+## What's new
+
+<!-- ### aws-cloud-provider
+
+* Support kube-control v2 schema
+
+### aws-k8s-storage
+
+* Support kube-control v2 schema -->
+
+### ceph-csi
+
+* [LP#2068524](https://bugs.launchpad.net/bugs/2068524)Reimagine ceph-csi charm with the reconciler pattern
+* Map ceph-csi to juju terraform syntax
+* Add `image-registry` configuration option
+* Create charm tolerations for ceph-rbd and cephfs deployments and daemonsets
+* Support alternate names for ceph-fs charm and associated storage class
+<!-- * Upgrade ceph upstream versions including 3.12 and 3.13 -->
+
+<!-- ### cinder-csi
+
+* Support kube-control v2 schema -->
+
+### etcd
+
+* Adding support from focal to noble
+* [LP#2053031](https://bugs.launchpad.net/charm-etcd/+bug/2053031) Adding tuning parameters
+
+### kubernetes-worker
+
+* [LP#2083925](https://bugs.launchpad.net/charm-kubernetes-worker/+bug/2083925)
+Add rules to access leases for nginx ingress
+
+### openstack-cloud-controller
+
+<!-- * Support kube-control v2 schema -->
+* Authorize the CCM to have CRD permissions
+* Map openstack-cloud-controller to juju terraform syntax
+
+### openstack-integrator
+
+* Map openstack-integrator to juju terraform syntax
+* lb-consumers now ignores default config lb-port if port mapping was provided
+
+## Component Versions
+
+### Charm/Addons pinned versions
+
+- kube-ovn ?????
+- calico ?????
+- cephcsi ?????
+- cinder-csi-plugin ?????
+- coredns ?????
+- ingress-nginx ?????
+- k8s-keystone-auth ?????
```

Comment:
[v1.30.0](https://github.com/canonical/keystone-k8s-auth-operator/blob/main/upstream/keystone_auth/version)

Well, we didn't really build a release for this.  Hmm... not sure what to do here with this one.

Reply:
I removed this altogether